README.md

```
@@ -5,7 +5,7 @@ Chanhyung's dotfiles, managed with [`chezmoi`](https://www.chezmoi.io/).
 Install them with:
 
 ```console
-$ chezmoi init kimchanhyung98
+$ chezmoi init --apply kimchanhyung98
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)